neuralmonkey/learning_utils.py

```diff
@@ -520,21 +565,21 @@ def _log_model_variables() -> None:
     trainable_vars = tf.trainable_variables()
     total_params = 0
 
-    logstr = "The model has {} trainable variables:\n\n".format(
-        len(trainable_vars))
+    logstr = "The model has {} trainable variables:\n\n"\
```

Comment:
omg to že to tak neni napsaný neznamená, že nechceme, aby to tak napsaný bylo v budoucnu.. Některý kusy kódu (např. tenhle learning_utils modul) jsou starší než výmysly o tom, jakej styl to bude mít. Podívej se do novějšího kódu, tam ten styl uvidíš.